packit_service/worker/checker/forges.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+
+from packit_service.worker.checker.abstract import Checker
+from packit_service.worker.mixin import GetIssueMixin
+
+logger = logging.getLogger(__name__)
+
+
+class PermissionOnForge(Checker, GetIssueMixin):
```

Comment:
Could we make this name a bit more specific? I feel like it's not really about permissions. Maybe something like `IsIssueInNotificationRepoChecker`? But that's just a suggestion